telematic_system/telematic_units/carma_cloud_bridge/cloud_nats_bridge/src/cloud_nats_bridge/cloud_nats_bridge.py

```diff
@@ -229,6 +227,9 @@ def xmlToJson(self, xmlString):
             self.logger.info("Error converting xml to json for: " + str(xmlString))
 
         return json_data
+            
```

Comment:
This is defining the queue_send function twice. For all of these application specific files, we want to keep what is there on the release candidate since that is tested code

Reply:
reverted what we have on the release/neon